spec/openapi.yaml

```diff
@@ -1525,6 +1525,31 @@ paths:
         
                 "We are authorized to use the Sender ID with a valid use case."
           - ```label``` name of the sender address.
+        ## Request a Sender Address from sub-accounts
+        Using API keys at the parent account level, you can request a sender address on behalf of a sub-account. To do this
+        include a header key ```Account``` with the sub-account ID as the value. For example:
+        ```Account: mySubAccount```
+
+        **Example request with Request a Sender Address from sub-accounts**
+      
+        ```plain
+        POST /v1/messaging/numbers/sender_address/requests HTTP/1.1
+        Host: api.messagemedia.com
+        Accept: application/json
+        Content-Type: application/json
+        Authorization: Basic dGhpc2lzYWtleTp0aGlzaXNhc2VjcmV0Zm9ybW1iYXNpY2F1dGhyZXN0YXBp
+        Account: SubAccount
+        {
+          "sender_address": "+61341234131",
+          "sender_address_type": "INTERNATIONAL",
+          "usage_type": "OWN_NUMBER",
+          "destination_countries": [
+              "AU"
+          ],
+          "reason": "I confirm that my business has a valid use case",
```

Comment:
The reason does not need to be in an escaped Json string?
```
"reason": "{\n  \"useCase\":\"AUSTRALIAN_GOVERNMENT_AGENCY_OR_ENTITY\",\n  \"description\":\"bal bla\",\n  \"email\":\"xample@email.com\",\n  \"australianGovernmentAgencyOrEntityName\":\"bla bla\",\n  \"statement\":\"We are authorised to use the Sender ID on behalf of [full entity name of sender] with a valid use case.\"\n}\n",
```

Reply:
Hi @archwind  To my understanding, it just needs to be String. 
Do you have any objection?

Reply:
Oh my bad. The escaped Json string is for ALPHA TAG requests only. String is fine for OWN NUMBER. Thanks.